keras/src/saving/saving_lib.py

```diff
@@ -35,6 +39,18 @@
 _ASSETS_DIRNAME = "assets"
 
 
+_MODEL_CARD_TEMPLATE = """
+---
+library_name: keras
+---
+
+This model has been uploaded using the Keras library and can be used with JAX, TensorFlow, and PyTorch backends.
```

Comment:
Please keep text lines under the character count limit for the codebase (this is `md` so line length doesn't matter, right?)

Reply:
Correct yes. Addressed in https://github.com/keras-team/keras/pull/19854/commits/d436eefb96a4c1fd5a44b104011ee276504f6782